### ISSUE TITLE
Meetup #1

### ISSUE THREAD

**Issue by betatim** [pydatazrh/meetup#5]
All things related to making the first meetup happen.

Topics: TBD

Location: TBD

Date: 15th or 22nd November (?)

**Comment by TitusCornelius** [comment]
Our first two topics will be:

on deep learning hardware and libraries, what you need to know to get serious about deep learning

on becoming an open source developer

there will be more info posted soon in the meetup group. The event is previsioned to happen sometime in early November and we still need a room.

**We really need to focus on finding a room now! Any help greatly appreciated**

**Comment by betatim** [comment]
I updated the top post with your comments. Should we already create a new event on the meetup group with this information? Not sure what the meetup etiquette is on this.

**Comment by TitusCornelius** [comment]
yes, I will announce the first meetup today.. From what I saw in other groups, it is quite common to post a meetup without a location and even a date. It seems quite common that the group participates in determining these.

**Comment by betatim** [comment]
The power of twitter: https://twitter.com/odi/status/790564177823825921 @metaodi 🚀

**Comment by betatim** [comment]
@TitusCornelius does Tim D have github? Would be good to chat with the speakers about talk length. My vote would go for 20(talking)+10(questions) each. What do you guys think?

**Comment by uweschmitt** [comment]
Maybe a colleague of mine and I can organize a room at ETH.... I'll know more tomorrow.

**Comment by metaodi** [comment]
@betatim I could offer the Arena at Liip on November 9. I made a tentative reservation of the room. Let me know if you want it. Or if you want a specific date, I can check again.

**Comment by betatim** [comment]
Awesome. 9 November sounds good to me. We probably need to run some kind of iterative algorithm between room, speakers etc to converge. Next iteration: speakers :)

**Comment by aktion-hip** [comment]
I could try to make a reservation for a nice room in the ETH main building, for Wednesday, 9. Nov. 19:30 to 22:00. It has room for 50 places.
If more places are needed, I could reserve a lecture room (160 places).
(PS: I’m the collegue @uweschmitt mentioned.)

**Comment by TitusCornelius** [comment]
Thanks everybody for all the support!

most important thing is to have a room in the first place, but it would be cool to have it so that people would want to stick around and that we could offer beers.

@metaodi how many people could fit in that room and could people stay for a beer (that we organize)?

@aktion-hip Thanks so much for offering! This also sounds really good.. In the ETH building I guess it would not be a good atmosphere/ not allowed to stick around afterwards with beer offered by us, or? I guess we could move things to the student bar afterwards.. Is it possible to book the room tentatively?

@betatim Tim is @TimDettmers

**Comment by betatim** [comment]
@timdettmers does 9 November work for you?

**Comment by betatim** [comment]
The room in the basement of Liip (if that is the room) is pretty big. Not sure about after party in terms of policy.

**Comment by aktion-hip** [comment]
I agree that the Liip room is better suited for the first PyData ZRH meetup, especially if it comes to build up the community.
The ETH main building would be good for presentations before a bigger audience.

**Comment by TitusCornelius** [comment]
@betatim Tim Dettmers told me that 9. November would work for him..

**Comment by betatim** [comment]
Then I would suggest we lock in the date and place and speakers 📸.

@TitusCornelius will you update the meetup page? Can we/do we want to link to this issue from there?

**Comment by TimDettmers** [comment]
9 November is good; 20 minutes + 10 minutes for the talk sounds reasonable. I would still need to fledge out the library part of the talk. I would talk a bit about deep learning libraries, and since it is pydata, focus mainly on the Python ones. I might also talk about one or the other utility library for deep learning or in general machine learning. However, overall I would focus more heavily on hardware, since that is my area of expertise. Probably around 12-14 minutes on hardware and and 6-8 minutes on libraries with 10 minutes for questions. 

What do you think?

**Comment by betatim** [comment]
Sounds perfect to me. Especially because I recently decided to treat myself to my first desktop machine in >5years in order to gain access to a good GPU 😄 

My feeling is that you either already know `tf`/`keras` or not in which case you might not be soo interested in this talk -> assume on the software side people are interested in more advanced little tips and tricks and not hearing about what cross-entropy loss is or that `keras` is a cool tool.

**Comment by betatim** [comment]
Does a 6.30pm start work for everyone?

**Comment by TitusCornelius** [comment]
@betatim: Did you hear anything from Jaime. He didn't answer to my mails..

We need to know if he can make it on the proposed date..

**Comment by betatim** [comment]
No news from him. I think we should make a move on announcing a firm time and place. If he can't make it we could try and find a replacement or move to drinks earlier/improvise. What do you think?

**Comment by TitusCornelius** [comment]
works for me!

@betatim: Thanks for updating the room on meetup

**Comment by betatim** [comment]
Let's do it. Worst comes to the worst I will use the free time to tell people about https://twitter.com/sentinel2bot :)

**Comment by TitusCornelius** [comment]
let's just go forward with it :)

**Comment by TitusCornelius** [comment]
I updated the date

**Comment by betatim** [comment]
@TitusCornelius should we plan a trip to a supermarket or similar for the day of the meetup to acquire some drinks/nibbles?

**Comment by TitusCornelius** [comment]
@betatim yes, we should.. I saw at the DevOps meetup, that they had two huge baguettes cut into many thin diagonal slices. I could make something like that and we get some chips, nachos and nuts? Any other suggestions? it looks like we won't be that many for starters, so I guess a keg of beer would be overkill for now.. We could get a palette of cheapish can beers.. that should do

Jaime contacted me and says that he didn't respond because he is swamped with work at the moment, but that he will make it to give his talk at the agreed upon date and place.

**Comment by betatim** [comment]
I will be in Zurich during the afternoon of the day of the meetup. Probably free after 4pm or 4.30pm. Does that work for you for a trip to a supermarket etc?

I will mention the meetup at the PyZurich meetup tomorrow in person and post a message on their board. I'll tweet about it again. Don't have any other smart ideas right now.

Good luck with the exam! 📚

**Comment by betatim** [comment]
Thanks everyone for the help with making this happen.

We had a tidy around after everyone was out. The empty cans are in the bins below the stairs. Is there a better place for them next time?
